cvat/apps/engine/tests/test_rest_api.py

```diff
@@ -956,6 +956,15 @@ def test_api_v2_projects_id_somebody(self):
     def test_api_v2_projects_id_no_auth(self):
         self._check_api_v2_projects_id(None)
 
+    def test_api_v2_projects_delete_project_data_after_delete_project(self):
+        for project in self.projects:
+            project_dir = project.get_dirname()
+            self.assertTrue(os.path.exists(project_dir))
+        self._check_api_v2_projects_id(self.admin)
+        for project in self.projects:
+            project_dir = project.get_dirname()
+            self.assertFalse(os.path.exists(project_dir))
```

Comment:
I believe #5632 was about deleting the data of the project's _tasks_, not just the project itself. So you should probably check that any task directories were deleted as well.

Reply:
Updated a test